src/sensai/featuregen.py

```diff
@@ -707,4 +785,4 @@ def flattenedFeatureGenerator(fgen: FeatureGenerator, columnsToFlatten: List[str
         return ChainedFeatureGenerator(fgen, flatteningGenerator)
     else:
         return ChainedFeatureGenerator(fgen,
```

Comment:
Because of this change in the signature the regression test fails. I could fix it by having a more flexible signature with the _flatten method (see DFTChain) but I would prefer to go into another direction and start having more consistent interfaces even if it means breaking changes. Here the inconsistency was between ChainedFeatureGenerator, MultiFeatureGenerator and ChainedDataFrameTransformer. There were some more, which I highlighted by TODOs. 

What do you think, Dominik?

Reply:
Definitely the flatten variant. Otherwise all persisted instances will be broken.

Reply:
However, I don't see how this change really creates more consistency in any way. Just because we happen to use a *list interface in one place doesn't mean we need to have it everywhere. There are many places where lists are expected, and it is often appropriate to make that list explicit (especially if the list isn't the only argument).
At the end of the day, you will immediately see what is required in your IDE anyway, so it's "difficult" to make a mistake. While I do think it's rather cumbersome, the flatten variant is OK for me, but I really don't think that adding square brackets is a big deal - and it's not really worse than having to add a star. What is more usable boils down to what you are usually likely to have. In the case of MultiFeatureGenerator, it was common to have the list, which is why it has the interface it has.

Reply:
Thanks for the lightning fast review! I will resort to the flatten variant to permit both interfaces then

Reply:
Done